doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -235,6 +235,9 @@ nRF Desktop
   * The release configuration for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target to enable the :ref:`nrf_desktop_failsafe` (see the :ref:`CONFIG_DESKTOP_FAILSAFE_ENABLE <config_desktop_app_options>` Kconfig option).
   * Enabled Link Time Optimization (:kconfig:option:`CONFIG_LTO` and :kconfig:option:`CONFIG_ISR_TABLES_LOCAL_DECLARATION`) by default for an nRF Desktop application image.
     LTO was also explicitly enabled in configurations of other images built by sysbuild (bootloader, network core image).
+  * Application configurations for nRF54L05, nRF54L10, and nRF54L15 SoCs to use Fast Pair PSA cryptography (:kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_PSA`).
+    Using PSA cryptography improves security and reduces memory footprint.
+    Also increased the size of the Bluetooth receiving thread stack (:kconfig::option:`CONFIG_BT_RX_STACK_SIZE`) to prevent stack overflows.
```

Comment:
```suggestion
    Also increased the size of the Bluetooth receiving thread stack (:kconfig:option:`CONFIG_BT_RX_STACK_SIZE`) to prevent stack overflows.
```